test/uk/gov/hmrc/agentuserclientdetails/support/FakeDesConnector.scala

```diff
@@ -39,10 +40,15 @@ case object FakeDesConnector extends DesConnector {
         )
       )
     )
-  def getTradingNameForMtdItId(
+  def getTradingDetailsForMtdItId(
     mtdbsa: MtdItId
-  )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[String]] =
-    Future.successful(Some("IT Client"))
+  )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[TradingDetails]] =
+    mtdbsa match {
+      case MtdItId("GK873907D") => Future.successful(Some(TradingDetails(Nino("GK873908D"), None)))
```

Comment:
btw - Mtdbsa has a different regex to a NINO, it's longer, but here is doesn't matter for the test.